api-report/purchases-js.api.json

```diff
@@ -2272,7 +2272,7 @@
         {
           "kind": "Enum",
           "canonicalReference": "@revenuecat/purchases-js!OfferingKeyword:enum",
-          "docComment": "/**\n * Keywords for identifying specific offerings in the {@link Purchases.getOfferings} method.\n */\n",
+          "docComment": "/**\n * Keywords for identifying specific offerings in the {@link Purchases.getOfferings} method.\n *\n * @public\n */\n",

```

Comment:
Was this addition to the doc intentional?

Reply:
yes. It was a check failure running tests and we updated it with the `@public` tag